DMX.Core.Api/Models/Foundations/Labs/Exceptions/LockedLabException.cs

```diff
@@ -0,0 +1,16 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace DMX.Core.Api.Models.Foundations.Labs.Exceptions
+{
+    public class LockedLabException : Xeption
+    {
+        public LockedLabException(Exception innerException)
+            : base(message: "Locked lab record exception. Please try again later.", innerException)
```

Comment:
call this "Locked lab record error occurred. Please try again later.

Reply:
These exception messages are meant for the muggles - muggles don't know what "exception" is :)